notebooks/code_samples/regression/quickstart_regression_full_suite.ipynb

```diff
@@ -41,14 +41,9 @@
    "source": [
     "## Before you begin\n",
     "\n",
-    "::: {.callout-tip}\n",
+    "<div class=\"alert alert-block alert-info\" style=\"background-color: #f7e4ee; color: #222425; border: 1px solid #222425;\">For access to all features available in this notebook, create a free ValidMind account.\n",
     "\n",
-    "### New to ValidMind?\n",
-    "\n",
-    "For access to all features available in this notebook, create a free ValidMind account.\n",
-    "\n",
-    "Signing up is FREE — [**Sign up now!**](https://app.prod.validmind.ai)\n",
-    ":::\n",
+    "Signing up is FREE — <a href=\"https://app.prod.validmind.ai\"><b>Sign up now</b></a></div>\n",
```

Comment:
Not for this pull request, but not all code samples have a table of contents, yet. This is an example — tracked via https://app.shortcut.com/validmind/story/4603/add-table-of-contents-to-remaining-notebooks.